src/io-methods/azure.js

```diff
@@ -0,0 +1,37 @@
+const azure = require("azure-storage");
+let blobService;
+
+function getBlobService() {
+  blobService = azure.createBlobService();
```

Comment:
Do we want to recreate a blob service every time? I don't know how Azure works -- it's probably fine but I thought I'd ask